superset-frontend/src/dashboard/components/SliceHeaderControls/index.tsx

```diff
@@ -39,6 +45,8 @@ import ModalTrigger from 'src/components/ModalTrigger';
 import Button from 'src/components/Button';
 import ViewQueryModal from 'src/explore/components/controls/ViewQueryModal';
 import { ResultsPaneOnDashboard } from 'src/explore/components/DataTablesPane';
+import Modal from 'src/components/Modal';
+import DrillDetailPane from 'src/dashboard/components/DrillDetailPane';
```

Comment:
```suggestion
import { DrillDetailPane } from 'src/dashboard/components/DrillDetailPane';
```

Reply:
I had it as the default export because it's the only export of the directory as the wiki suggests:

> Default VS named exports: As recommended by [Typescript](https://www.typescriptlang.org/docs/handbook/modules.html), “If a module’s primary purpose is to house one specific export, then you should consider exporting it as a default export. This makes both importing and actually using the import a little easier”. If you’re exporting multiple objects, use named exports instead.

That ok?